Code/RDGeneral/types.cpp

```diff
@@ -29,6 +29,7 @@ const std::string _2DConf = "_2DConf";
 const std::string _3DConf = "_3DConf";
 const std::string _AtomID = "_AtomID";
 const std::string _BondsPotentialStereo = "_BondsPotentialStereo";
+const std::string _CanonicalAtomRank = "_CanonicalAtomRank";
```

Comment:
given that this is used only for stereochemistry, it's probably best to rename it to `_ChiralAtomRank`

Reply:
OK